package.json

```diff
@@ -16,12 +16,12 @@
   "bugs": {
     "url": "https://github.com/erlang-ls/vscode/issues"
   },
-  "main": "./client/out/client.js",
+  "main": "./client/out/extension.js",
   "scripts": {
     "package": "vsce package",
     "test": "echo \"Error: no test specified\" && exit 1",
     "vscode:prepublish": "cd client && npm run compile",
-    "compile": "cd erlang_ls && rebar3 escriptize && cd ../client && npm run compile",
+    "compile": "cd erlang_ls && make && cd ../client && npm run compile",
```

Comment:
We can probably skip the dependency on `make` here and add the _escriptization_ of the DAP explicitly. Can be done as a follow up, though.